Hand the PCS the QCA8084 reset line so it can reset the package again
after the early UNIPHY0 bring-up (vendor order).
--- a/target/linux/qualcommbe/dts/ipq9574-be7000.dts
+++ b/target/linux/qualcommbe/dts/ipq9574-be7000.dts
@@ -987,6 +987,7 @@
 
 &pcs0 {
 	qcom,early-10g-qxgmii;
+	qcom,phy-package-reset-gpios = <&tlmm 60 GPIO_ACTIVE_LOW>;
 };
 
 &pcs2 {
